content/en/docs/Reference/file-based-catalogs.md

```diff
@@ -82,28 +82,28 @@ they have the same rules for [patterns](https://git-scm.com/docs/gitignore#_patt
 > ```
 
 
-Index maintainers have the flexibility to chose their desired layout, but the OLM team recommends storing each package's
+Catalog maintainers have the flexibility to chose their desired layout, but the OLM team recommends storing each package's
 file-based catalog blobs in separate sub-directories. Each individual file can be either JSON or YAML -- it is not
-necessary for every file in an index to use the same format.
+necessary for every file in a Catalog to use the same format.
 
-This layout has the property that each sub-directory in the directory hierarchy is a self-contained index, which makes
-index composition, discovery, and navigation as simple as trivial filesystem operations.
+This layout has the property that each sub-directory in the directory hierarchy is a self-contained Catalog, which makes
+Catalog composition, discovery, and navigation as simple as trivial filesystem operations.
 
 > **Basic recommended structure**
 > ```
-> index
+> catalog
 > ├── pkgA
-> │   └── index.yaml
+> │   └── operator.yaml
```

Comment:
My `opm migrate` PR currently names this `catalog.yaml`. I don't have a strong opinion, but it would be good for there to be alignment on this.

Reply:
@joelanford is `operator.yaml` more suitable than `catalog.yaml`? 

Also, all of the new opm commands call the catalog `index` (also documented in this site). Maybe we should change those too while we're at it?

Reply:
How about `package.yaml` since the directory represents a package and since its just a single file, it must be the entire package's worth of metadata.

I'm hedging a little bit based on the fact that some packages may just be API definitions without an associated operator definition/deployment.

Reply:
I guess `package.yaml` works, the only reason I was trying to avoid that is because historically `package.yaml` meant something else for the community. Eg: https://github.com/operator-framework/community-operators/blob/master/community-operators/etcd/etcd.package.yaml